release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -611,6 +611,34 @@ Previously, the `thinPoolConfig.overprovisionRatio` field in the `LVMCluster` cu
 [id="ocp-release-notes-web-console_{context}"]
 === Web console
 
+[id="ocp-4-18-administrator-perspective_{context}"]
+==== Administrator perspective
+
+This release introduces the following updates to the *Administrator* perspective of the web console:
+
+* A new setting for hiding the *Getting Started Resources* card on the *Overview* page allowing for maximum use of the dashboard.
+* A *Start Job* option was added to the *List* and *Details* pages, so you can start individual CronJobs manually directly in the web console without having to use the `oc` CLI.
+* The *Import YAML* button in the masthead is now a *Quick Create* button, and to allow rapid deployment of workloads, you can use YAML, Git, or a container image.
+* You can build your own generative-AI chat bot with a chat bot sample. The generative-AI chat bot sample is deployed with Helm and includes a full CI/CD pipeline. You can also run this sample on your cluster with no CPUs.
+* You can import YAML into the console using {ols}.
```

Comment:
sorry, where are these features coming from?

Reply:
thanks! 
Shall we move these lines to `Developer Perspective` or just keep them here? 
@sanketpathak Could you help review the contents?